packages/x-date-pickers/src/MonthCalendar/MonthCalendar.tsx

```diff
@@ -61,7 +61,7 @@ export interface MonthCalendarProps<TDate>
   disableHighlightToday?: boolean;
   onMonthFocus?: (month: number) => void;
   hasFocus?: boolean;
-  onFocusedViewChange?: (newHasFocus: boolean) => void;
+  onFocusedViewChange?: (hasFocus: boolean) => void;
```

Comment:
With the "new" it sounds more obvious that the function has to be call when the focus is modified. Not a strong opinion

Reply:
My main rational here was that on `onChange` we are always calling it `value` not `newValue` in the interface

(example from the `base` package)
```tsx
onChange?: (event: React.SyntheticEvent, value: number | string) => void;
```